scripts/postbuild.ts

```diff
@@ -0,0 +1,61 @@
+/*
+  This script prepares the central `build` directory for NPM package creation.
+  It first copies all non-code files into the `build` directory, including `package.json`, which
+  is edited to adjust entry point paths. These corrections are performed so that the paths align with
+  the directory structure inside `build`.
+*/
+
+import * as fs from 'fs';
+
+import * as path from 'path';
+
+const BUILD_DIR = 'build';
+const ASSETS = ['README.md', 'LICENSE', 'package.json', '.npmignore'];
+const ENTRY_POINTS = ['main', 'module', 'types'];
+
+// check if build dir exists
+try {
+  if (!fs.existsSync(path.resolve(BUILD_DIR))) {
+    console.error(`Directory ${BUILD_DIR} DOES NOT exist`);
+    console.error("This script should only be executed after you've run `yarn build`.");
+    process.exit(1);
+  }
+} catch (error) {
+  console.error(`Error while looking up directory ${BUILD_DIR}`);
```

Comment:
Should these catch statements also make the process exit? Ditto with the ones below.

Reply:
Good catch, thanks